src/emulator/gui/src/gui.cpp

```diff
@@ -142,9 +142,9 @@ static void init_font(State &gui) {
     gui.normal_font = io.Fonts->AddFontFromMemoryTTF(gui.font_data.data(), static_cast<int>(font_file_size), 16, &font_config, io.Fonts->GetGlyphRangesJapanese());
 }
 
-static void init_background(State &gui, const std::string &image_path) {
+void init_background(HostState &host, const std::string &image_path) {
     if (!fs::exists(image_path)) {
-        LOG_INFO("Invalid background file path {}.", image_path);
+        LOG_WARN("Image not found from {}, check your image path set.", image_path);
```

Comment:
```suggestion
        LOG_WARN("Image doesn't exist {}.", image_path);
```

Simplify the statement